### PR TITLE
improved styling of the notification dropdown

### PR DESCRIPTION
**Pull request by haukkagu**
* Moved the icons to the top of each notification
* Added a separator between each notification
* Decreased line height from 1.8 to 1.5
* Increased the height of the dropdown by 100 pixels

Screenshot:
notification4

### CODE REVIEW

File: app/components/HeaderNotifications/HeaderNotifications.css [hunk 1]
```diff
@@ -22,10 +22,13 @@ html[data-theme='dark'] .notifications {
 
 .notification {
   border-bottom: 1px solid lightgray;
-  padding: 15px;
+  padding-top: 15px;
+  padding-right: 15px;
+  padding-left: 15px;
```
Comment (ivarnakken) [inline app/components/HeaderNotifications/HeaderNotifications.css:27]: This is the same as `padding: 15px 15px 0`

Comment (LudvigHz) [inline app/components/HeaderNotifications/HeaderNotifications.css:27]: You can actually rewrite this to
```suggestion
  padding: 15px 15px 0 15px;
```

See https://developer.mozilla.org/en-US/docs/Web/CSS/padding#syntax

File: app/components/HeaderNotifications/HeaderNotifications.css [hunk 2]
```diff
@@ -41,9 +44,10 @@ html[data-theme='dark'] .notifications {
 }
 
 .innerNotification {
+  border-bottom: 1px solid var(--color-mono-gray-3);
+  padding-bottom: 15px;
```
Comment (ivarnakken) [inline app/components/HeaderNotifications/HeaderNotifications.css:48]: Veeery picky, but nice to get in early: please move color and "styling" css to the bottom, and let position/display/size lines come first.

**Review by ivarnakken — CHANGES_REQUESTED** [review]
This looks much better! 💯 

But, it's kinda annoying that the border line is not as long as the text - any way you can change that?

File: app/components/HeaderNotifications/HeaderNotifications.css (hunk 1 above)
Comment (haukkagu) [reply app/components/HeaderNotifications/HeaderNotifications.css]: fixed

File: app/components/HeaderNotifications/HeaderNotifications.css (hunk 2 above)
Comment (haukkagu) [reply app/components/HeaderNotifications/HeaderNotifications.css]: fixed

File: app/components/HeaderNotifications/HeaderNotifications.css (hunk 1 above)
Comment (haukkagu) [reply app/components/HeaderNotifications/HeaderNotifications.css]: fixed

**Review by LudvigHz — APPROVED** [review]

**Pull request merged**